packages/expo-video/src/resolveAssetSource.web.ts

```diff
@@ -0,0 +1,17 @@
+import { getAssetByID } from '@react-native/assets-registry/registry';
```

Comment:
@byCedric we shouldn't import it, right? It's not a dependency and don't want it to be direct.

Reply:
You are right, for this to work we will have to add  `"@react-native/assets-registry": "*"` as a `peerDependency` for `expo-video`